actpool/actqueue.go

```diff
@@ -106,6 +109,30 @@ func (q *actQueue) Put(act *action.SealedEnvelope) error {
 		if nonce < q.pendingNonce && act.GasFeeCap().Cmp(actInPool.GasFeeCap()) != 1 {
 			return action.ErrReplaceUnderpriced
 		}
+		// 2x bumps in gas price are allowed for blob tx
+		isPrevBlobTx, isBlobTx := len(actInPool.BlobHashes()) > 0, len(act.BlobHashes()) > 0
+		if isPrevBlobTx {
+			if !isBlobTx {
+				return errors.Wrap(action.ErrReplaceUnderpriced, "blob tx can only replace blob tx")
+			}
+			var (
+				prevBlobGasFeeCap = actInPool.BlobGasFeeCap()
+				blobGasFeeCap     = act.BlobGasFeeCap()
+
+				priceBump        = big.NewInt(2)
+				minGasFeeCap     = new(big.Int).Mul(actInPool.GasFeeCap(), priceBump)
+				minGasTipCap     = new(big.Int).Mul(actInPool.GasTipCap(), priceBump)
+				minBlobGasFeeCap = new(big.Int).Mul(prevBlobGasFeeCap, priceBump)
```

Comment:
no need to define `prevBlobGasFeeCap` at L119, directly use `actInPool.BlobGasFeeCap()` here?